.changeset/mean-owls-own.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": patch
+---
+
+feat: Add validation to the `name` field in configuration.
+The validation will warn users that the field can only be "type string,
+alphanumeric and lowercase with dashes only" using the same RegEx as the backend
```

Comment:
Are we intentionally not mentioning underscores, even though they are "valid" right now.

Reply:
Good question, I could add that too